### PR TITLE
feat: add and configure auto library

### PR DESCRIPTION
**Pull request by tinytim84**
I thought I'd do this as a branch in the end so you can better see what I've changed.  This seemed to work correctly on the dry run, but I may have still missed some bits.  There's definitely a bit more to be done if we want to be able to skip release by using a label.

### CODE REVIEW

File: .github/labels.yml [hunk 1]
```diff
@@ -32,3 +29,30 @@
 - name: enhancement
   description: adds a new feature or improves on an existing one
   color: ad12cc
+- name: dependencies
+  description: Update one or more dependencies version
+  color: 8732bc
```
Comment (AndrewLeedham) [inline .github/labels.yml]: Can we keep the original colour and description for this label.

File: .github/labels.yml [hunk 2]
```diff
@@ -32,3 +29,30 @@
...
+  color: 696969
+- name: documentation
+  description: Changes only affect the documentation
+  color: cfd3d7
+- name: tests
+  description: Add or improve existing tests
+  color: ffd3cc
+- name: performance
+  description: Improve performance of an existing feature
+  color: f4b2d8
```
Comment (AndrewLeedham) [inline .github/labels.yml]: image
Can you tweak the colours a little, I would say the following are too close:
- bug <> minor
- cron <> release
- blocked <> patch

**Review by AndrewLeedham — CHANGES_REQUESTED** [review]

File: .github/labels.yml (hunk 2 above)
Comment (tinytim84) [reply .github/labels.yml]: Sure, they were just the defaults.

Comment (tinytim84) [reply .github/labels.yml]: I've updated, does it pull the labels from the default branch? i.e. next?

Comment (AndrewLeedham) [reply .github/labels.yml]: No it runs the `sync-labels.yml` workflow on merge into release

File: .github/labels.yml (hunk 1 above)
Comment (AndrewLeedham) [reply .github/labels.yml]: :eyes:

Comment (tinytim84) [reply .github/labels.yml]: Sorry, put the colour back but not the description 🤦

Comment (tinytim84) [reply .github/labels.yml]: done now.

**Review by AndrewLeedham — APPROVED** [review]
LGTM I'll merge it into release on Monday

**Pull request merged**